start-server.sh

```diff
@@ -15,14 +15,28 @@
 # limitations under the License.
 
 #provide the path to Node directory before starting the run
-#NODE_PATH=/opt/local/node-v6.10.0-linux-x64/
-if [ "x$NODE_PATH" == "x" ]; then
-	echo "Node path is not set"
+export PATH=/opt/local/node-v6.10.0-linux-x64/bin:$PATH
+
+IMAGE_NAME=""  #if not set, workload uses image.jpeg by default
+
+npm_ret=`which npm`
```

Comment:
Apparently, you are not supposed to use `which` to check if program exists

https://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script/677212#677212